### PR TITLE
bbbconfgen: add #check option

### PR DESCRIPTION
**Pull request by Stadicus**
The `#check` function allows to drop a line if a key is set to `0`, or not set at all:
```
{{ key #check }}
```

The option can be set anywhere on the line and works together with all other options.

### CODE REVIEW

File: tools/bbbconfgen/test.conf [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# network
```
Comment (0xB10C) [inline tools/bbbconfgen/test.conf]: Should this file be added?

File: tools/bbbconfgen/test/bitcoin-template.conf [hunk 2]
```diff
@@ -3,6 +3,7 @@ mainnet={{ bitcoind:mainnet }}
 testnet={{ bitcoind:testnet #default: 0 }}
 rpcconnect={{ bitcoind:rpcconnect }}
 dbcache={{ bitcoind:dbcache #default: 300 }}
-seednode={{ bitcoind:seednode:1 #rmLine }}
-seednode={{ bitcoind:seednode:2 #rmLine }}
-seednode={{ bitcoind:seednode:3 #rmLine }}
+{{ bitcoind:testnet #check }}printtoconsole=1
```
Comment (0xB10C) [inline tools/bbbconfgen/test/bitcoin-template.conf]: Switch it around i.e.: `printtoconsole=1 {{ bitcoind:testnet #check }}`. Easier to read and consistent with the `seednode` entries below.

File: tools/bbbconfgen/test.conf (hunk 1 above)
Comment (Stadicus) [reply tools/bbbconfgen/test.conf]: removed

File: tools/bbbconfgen/test/bitcoin-template.conf (hunk 2 above)
Comment (Stadicus) [reply tools/bbbconfgen/test/bitcoin-template.conf]: I wanted to explicitly have both options present as an example. But I agree, in the new config files it's like you suggest always, so maybe that's not necessary.

Comment (Stadicus) [reply tools/bbbconfgen/test/bitcoin-template.conf]: adjusted to your suggestion

**Pull request merged**